CMS/static/scss/global.scss

```diff
@@ -380,3 +380,14 @@ table {
     outline: none;
 }
 
+@media (min-width: 768px) {
+  div.header-block {
+    min-width: 515px;
```

Comment:
Is there a reason for changing this to `min-width`?
It will mean the box shape will change based on the text inside.

![image](https://user-images.githubusercontent.com/6823289/118466963-46743700-b6fb-11eb-8d64-796c895ba728.png)

Reply:
(otherwise looks good though)

Reply:
We made the change because when zooming in the text inside the box, the text gets cut out across the site and we thought this might be an issue, even though this was not highlighted in the accessibility report. 

![Screenshot 2021-05-17 at 11 13 54](https://user-images.githubusercontent.com/70764326/118473983-7b37bc80-b702-11eb-97da-3b40e99d16f3.png)

Issue: supporting material:

https://drive.google.com/drive/folders/1OuUIEBzwzO5BAZRr_b2un0lfbF3bekWm

Reply:
Ah, I get it - it is quite a large design change though and I think they have done things like adapt the background pictures to fit with a fixed-width title box - in other circumstances this would be one to take to the Product Owner but given it isn't part of the current ask I think stick with fixed-width